symplyphysics/laws/kinematic/vector/velocity_of_transfer_between_reference_frames.py

```diff
@@ -0,0 +1,64 @@
+from symplyphysics import (
+    units,
+    angle_type,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    add_cartesian_vectors,
+    scale_vector,
+    cross_cartesian_vectors,
+)
+
+# Description
+## Suppose two reference frames, one of which is fixed (S) and the other one is moving (S'). The movement of
+## a body stationary in moving frame S' due to the movement of the frame itself is called transfer movement.
+## The velocity related to such movement can be called transfer velocity.
+
+# Law: v_tr = v_0 + cross(w, r)
+## v_tr - vector of transfer velocity
+## v_0 - vector of velocity of moving origin O' of frame S' relative to fixed frame S
+## w - vector of angular velocity related to rotation of moving frame S' about instantaneous axis
```

Comment:
Why does it rotate? In the description only motion was mentioned.

Reply:
Motion implies rotation

Reply:
Yes, but let's not place too much burden on a reader. Mention that we also allow rotational motion in this formula.